students/TinaB/lesson09/mailroom.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
+
+"""Updated Mailroom Script with classes
+$ chmod +x mailroom.py
+"""
+
+import datetime
+import os
+
+
+class Donor:
+
+    def __init__(self, firstname, lastname, donations=None):
+        self._firstname = firstname
+        self._lastname = lastname
+        if donations is None:
+            self._donations = []
+        else:
+            self._donations = donations
+
+    @property
+    def fullname(self):
+        """returns string with fullname of donor"""
+        return f"{self._firstname} {self._lastname}"
+
+    @property
+    def firstname(self):
+        return self._firstname
+
+    @property
+    def lastname(self):
+        return self._lastname
+
+    @property
+    def donations(self):
+        return self._donations
```

Comment:
You don't need to use `@property`s if you aren't adding any custom logic or processing to the attribute